src/components/calcite-input/calcite-input.scss

```diff
@@ -119,87 +145,85 @@
 //focus
 :host textarea,
 :host input {
-  @include focus-style-base();
+  @apply focus-base;
 }
 :host textarea:focus,
 :host input:focus {
-  @include focus-style-inset();
+  @apply focus-inset;
 }
 
 // states
 :host {
   & input,
   & textarea {
-    color: var(--calcite-ui-text-1);
-    border: 1px solid var(--calcite-ui-border-1);
+    @apply text-color-1 
+      border 
+      border-solid 
+      border-color-input;
 
     &::placeholder,
     &:-ms-input-placeholder,
     &::-ms-input-placeholder {
-      color: var(--calcite-ui-text-3);
-      font-weight: 400;
+      @apply text-color-3 font-normal;
     }
   }
 
   & input:focus,
   & textarea:focus {
-    border-color: var(--calcite-ui-brand);
-    color: var(--calcite-ui-text-1);
+    @apply border-color-brand text-color-1;
   }
   & input[readonly],
   & textarea[readonly] {
-    background-color: var(--calcite-ui-background);
+    @apply bg-background;
   }
   & input[readonly]:focus,
   & textarea[readonly]:focus {
-    color: var(--calcite-ui-text-1);
+    @apply text-color-1;
   }
   & calcite-icon {
```

Comment:
Worked with Bryan on this, we'll likely need to revisit this component to better construct it and use tailwind.